erpnext_germany/erpnext_germany/doctype/business_letter/business_letter.json

```diff
@@ -182,12 +183,22 @@
    "fieldname": "date",
    "fieldtype": "Date",
    "label": "Date"
+  },
+  {
+   "fieldname": "company",
+   "fieldtype": "Link",
+   "in_list_view": 1,
```

Comment:
```suggestion
```

Can we get rid of this? It's not a common pattern in ERPNext to show the company in the list view. And it can easily be customized, if needed.